libs/output-mapping/src/Writer/Helper/TableColumnsHelper.php

```diff
@@ -28,9 +28,10 @@ public static function addMissingColumns(
             return;
         }
 
-        $defaultBaseTypeValue = $currentTableInfo['isTyped'] === true ? BaseType::STRING : null;
```

Comment:
Tohle bude zbytečný. V KBC už je PR na backfiil https://github.com/keboola/connection/pull/4351

https://keboola.slack.com/archives/CFVRE56UA/p1682334733867809

Reply:
Tak asi nakonec bude zbytečný celý tenhle PR, když to byl problém jen jedné tabulky?

Reply:
To asi nebyl problem jedne tabulky, ale jednoho backendu :)
Ale po tom backfillu uz tohle nastat nemuze ze by `isTyped` nebylo true :)
Takze jo, tohle se nemusi resit

Reply:
Kazdopadne teda tohle odebrat.-